src/lib/web-server.js

```diff
@@ -1,16 +1,21 @@
 import express from 'express'
 import Settings from './settings.js';
+import Status from './status.js';
```

Comment:
Can we call this file web-status, matches scheme in #7 ?

I suspect it will get worked on to add other data that is highly specific to the Web API, so it would be good to be clear this is what we are doing to differentiate from other status stuff we may do later.

Actually status seems generic (the status of what?) and maybe sounds like others things we would add later (like an Prometheus end point). Can't immediately think of a specific name tho. Maybe just "publishers"?

Reply:
Yeah agreed, `publisher-status.js` ?

Reply:
Yes! But web-publisher-status.js ok?